server/db/index.js

```diff
@@ -23,11 +23,48 @@ connection.connect((err) => {
   }
 });
 
+// TEST QUERY
 const test = () => {
   const mysqlQuery = 'SELECT * FROM User;';
   return query(mysqlQuery);
 };
 
+// USER QUERIES
+const getUser = (email) => {
+  const mysqlQuery = 'SELECT * FROM User WHERE Email = ?;';
+  return query(mysqlQuery, [email]);
+};
+const postUser = ({
+  username, nameFirst, nameLast, phoneNumber, email,
+}) => {
+  const mysqlQuery = 'INSERT INTO User VALUES(null, ?, ?, ?, ?, ?);';
+  return query(mysqlQuery, [username, nameFirst, nameLast, phoneNumber, email]);
+};
+
+// CRAWL QUERIES
+const getCrawl = (idCreator) => {
+  const mysqlQuery = 'SELECT * FROM User WHERE Email = ?;';
```

Comment:
this function takes in an id as a parameter? or an email? right now it's saying SELECT * FROM User WHERE Email = idCreator
also, after we select that user, we need to use the user_crawl join table to select the crawls that have Id_Profile equal to the selected user's Id